### ISSUE TITLE
Are recipes supported?

### ISSUE THREAD

**Issue by accountnumber3** [codetheweb/MMM-AnyList#10]
Any list has an amazing recipe engine. I want to use the entire screen to show the ingredients and steps.

**Comment by codetheweb** [comment]
Recipes are not currently supported. You'd have to add recipe support to the underlying [anylist](https://github.com/codetheweb/anylist) module first.

I have no plans to do this since it's not personally useful, but I'm happy to accept PRs.

**Comment by accountnumber3** [comment]
That's fair. I'm more percussive-maintenance than developer. I _reeeeally_ need this functionality but I don't think I'm capable of it. Can I buy you a subscription or something? Can you give me some hints on where to start?

**Comment by gpoprox** [comment]
@codetheweb, @accountnumber3   i haven't looked, but are recipes something that anylist already handles? I could look into this. i think i might look into the other request first since it seems easier :D

**Comment by accountnumber3** [comment]
@gpoprox  yes, the AnyList product has a really nice interface for recipes. The MMM-AnyList module doesn't support it.

**Comment by gpoprox** [comment]
@accountnumber3 cool. I look into it and see if we can make something work

**Comment by accountnumber3** [comment]
@gpoprox awesome, tyvm!

This concept is the entire reason I got into MM in the first place. Happy to test anything.
